.travis.yml

```diff
@@ -37,38 +38,45 @@ jobs:
     # a scheduled task to build the weekly dev build on the main branch. The
     # weekly frequency is meant to avoid depleting the Travis CI credits too
     # fast.
-    - python: 3.7
-      os: linux
+    - os: linux
       arch: arm64-graviton2
       dist: focal
-      virt: lxd
+      virt: vm
```

Comment:
I think I was hitting the [storage limit of 18gb](https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system) for using lxd when downloading the docker image. Using `virt: vm` allows for 50GB of storage.

Reply:
Any noticeable impact on speed?

Reply:
When compared to the [most recent build weekly ARM build](https://app.travis-ci.com/github/scikit-learn/scikit-learn/builds/241010626). Setting `vm` is [a bit slower](https://app.travis-ci.com/github/scikit-learn/scikit-learn/builds/242108738).

For reference, [this was the memory error](https://app.travis-ci.com/github/scikit-learn/scikit-learn/jobs/546993596) I was seeing.